.aspect/workflows/config.yaml

```diff
@@ -1,138 +1,57 @@
 ---
+queue: aspect-small
 workspaces:
     .:
         icon: js
         label: rules_js
-    e2e/bzlmod:
-        icon: bazel
         tasks:
             format:
-                without: true
+                soft_fail: false
             buildifier:
-                without: true
+            test:
+                queue: aspect-default
+    e2e/bzlmod:
+        icon: bazel
     e2e/js_image_docker:
         icon: docker
-        tasks:
-            format:
```

Comment:
Do these get run at all now for the e2e tests?

Reply:
No. I found a better pattern for it that doesn't require the negations

Reply:
Oh I see 👍, I think I misread that